core/docker/default/etc/jvm.config

```diff
@@ -12,3 +12,8 @@
 -XX:PerBytecodeRecompilationCutoff=10000
 -Djdk.attach.allowAttachSelf=true
 -Djdk.nio.maxCachedBufferSize=2000000
+# We enable -XX:+UnlockDiagnosticVMOptions and -XX:+UseAESCTRIntrinsics to use a backport introduced by https://github.com/openjdk/jdk11u-dev/pull/410.
+# This gives a significant performance improvment of AES CTR/GCM enryption for ARM64 architecture.
+# TODO: remove -XX:+UnlockDiagnosticVMOptions and -XX:+UseAESCTRIntrinsics after upgrading to at least Java 17: https://github.com/trinodb/trino/issues/12253
```

Comment:
why Java 17 and not 18? Also I think it might be enabled by default in 11 too eventually

Reply:
You right - edited.